assignments/callbacks.js

```diff
@@ -20,33 +20,60 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   });
 
 */
-
+const log = function(param){
+  console.log(param);
+}
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, log);
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
+last(items, log)
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
+sumNums(2,7,log);
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
+multiplyNums(5,5,log);
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
+  for(let i = 0; i<list.length; i++){
+    if(list[i] == item){
+      return cb(true);
+    } else{
+      return cb(false);
+    }
+  }
   // Pass true to the callback if it is, otherwise pass false.
 }
+contains("Matt", ["Matt", "Nick", "Andrew"], log);
 
 /* STRETCH PROBLEM */
 
-function removeDuplicates(array, cb) {
+// Need to work through this more, was researching online and found this function, need to understand it deeper.
```

Comment:
I would suggest looking at the solution and trying to crack it, then make your own to your understanding.